packages/lib/services/interop/InteropService.ts

```diff
@@ -58,6 +58,14 @@ export default class InteropService {
 					isNoteArchive: false, // Tells whether the file can contain multiple notes (eg. Enex or Jex format)
 					description: _('Markdown'),
 				},
+				{
+					...defaultImportExportModule(ModuleType.Importer),
+					format: 'yaml',
+					fileExtensions: ['md', 'markdown', 'txt', 'html'],
+					sources: [FileSystemItem.File, FileSystemItem.Directory],
+					isNoteArchive: false, // Tells whether the file can contain multiple notes (eg. Enex or Jex format)
+					description: _('Markdown + YAML'),
```

Comment:
Should we call that "Markdown + Front Matter"? As it seems to return good results on Google when searching for it.

Reply:
Good idea, I was thinking about this yesterday and convinced myself that calling it YAML is bad, but I couldn't come up with anything better. I like "Markdown + Front Matter".

Reply:
Although I guess even Front Matter is a technical term. Maybe it should be "Markdown + Metadata"?

Reply:
I see what you mean but I think it makes sense to use a googlable term in this case, and "markdown metadata" doesn't return much, while "markdown front matter" returns exactly the format you've created, with documentation, tools and so on to process it

We can add some documentation if there's some confusion but if the term can be searched online it's somehow self-documenting too.

Reply:
I don't like "Front Matter" because it's basically gibberish. But you're right, i tried searching it and there's a ton of resources to help introduce written for different levels of technical ability. It doesn't get much better than that. I've updated it now.